cogment_verse/specs/data.proto

```diff
@@ -58,6 +58,10 @@ message Observation {
   optional nd_array.Array action_mask = 3;
   optional bytes rendered_frame = 4;
   repeated string overridden_players = 5;  // list of players that provided an action that was overriden during the last tick
+  optional string game_player_name = 6;
+  optional bool feedback_required = 7;
+  optional int32 action_value = 8;
+  optional int32 step = 9;
```

Comment:
?

Reply:
I replace `step` by `tickId`. The turn-by-turn procedure is implemented to handle human feedback so that the field `feedback_required` is added. We also provide the essential info  such as `game_player_name` representing the name of last player is playing and its action `action_value`. Human will rely on this info to evaluate the actor performance